.github/workflows/ci.yml

```diff
@@ -18,11 +18,9 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-22.04, ubuntu-20.04]
-        elixir_version: [1.12.3, 1.13.3, 1.14.1]
-        otp_version: [24, 25]
-        exclude:
-          - otp_version: 25
-            elixir_version: 1.12.3
+        elixir_version: [1.13, 1.14, 1.15, 1.16]
```

Comment:
I don t think there is a release fir 1.13 and OTP 26,  we will need to exclude that from the matrix (as I did for OTP25 and 1.12